### PR TITLE
OKTA-77056

### PR DESCRIPTION
**Pull request by tthompson-okta**
Make a few corrections to the ID token examples and client registration guide.

@mystiberry-okta @hassenkaraa-okta @slee-okta @joelfranusic-okta @karlmcguinness-okta @federations-okta

### CODE REVIEW

File: _source/docs/guides/social_authentication.md [hunk 1]
```diff
@@ -73,13 +73,15 @@ Advantages of using Okta Social Authentication include:
     
     **Note:** This URL will look similar to this one:
     `https://example.okta.com/oauth2/v1/authorize?idp=0oa0bcde12fghiJkl3m4`
-6.  Add an OAuth client via the Okta API
+6.  Add an OAuth client via the App Integration Wizard or via the [Okta API](../resources/oauth-clients.html)
     
-    For now, you will need to do this by making HTTP requests
-    directly against the Okta API. We suggest using Postman to do
+    Registering an OAuth client via the App Integration Wizard is done using the "Create New App" button on the Applications dashboard.
```
Comment (cwu-okta) [inline _source/docs/guides/social_authentication.md]: Nit: do we call it Applications dashboard in other documentation? I didn't think it had a name. We typically only say how you navigate to a specific place, as shown in the app wizard product doc: https://c.na10.visual.force.com/apex/knowledge_detail?id=kA0F0000000U32t

**Comment by cwu-okta** [comment]
@eklein-okta who is covering for Mysti while she is out on PTO

File: _source/docs/guides/social_authentication.md (hunk 1 above)
Comment (tthompson-okta) [inline _source/docs/guides/social_authentication.md]: I'm open to suggestions on this one.

Comment (cwu-okta) [inline _source/docs/guides/social_authentication.md]: Perhaps:
``` Register an OAuth Client by navigating to the Administrator Dashboard, go to Applications, then click Add App. Click Create New App to launch the Application Integration Wizard```

Tech doc probably has a better suggestion than mine.

**Comment by tthompson-okta** [comment]
Feedback has been addressed.

**Comment by cwu-okta** [comment]
:ramen:

**Pull request merged**